packages/components/src/date-time/stories/time.story.tsx

```diff
@@ -49,3 +51,16 @@ const Template: StoryFn< typeof TimePicker > = ( {
 };
 
 export const Default: StoryFn< typeof TimePicker > = Template.bind( {} );
+
+const TimeInputTemplate: StoryFn< typeof TimePicker.TimeInput > = ( args ) => {
+	return <TimePicker.TimeInput { ...args } />;
```

Comment:
Felt a bit odd to me that the `currentTime` doesn't seem to work for the `TimeInput` story, while it does for the default `TimePicker` one:


https://github.com/WordPress/gutenberg/assets/8436925/29a73d96-958a-4c28-a8b2-35b66118b0f9

Reply:
You're right, I should've hidden the controls for that story (1bb3f2f).